### PR TITLE
Tried to clarify the nested array prohibition

### PR DESCRIPTION
**Pull request by PeterBell**
to make it more beginner readable but still ensure nobody actually does create a nested array

Would love :eyes: @pletcher and @AnnJohn

### CODE REVIEW

**Comment by PeterBell** [comment]
Looks like the linter is unhappy:
LICENSE.md file results
- missing LICENSE.md
- invalid or missing license content
  CONTRIBUTING.md file results
- invalid or missing contributing content
- missing CONTRIBUTING.md

I'm not going to mess up this PR by adding those. Let me know @pletcher and @AnnJohn whether we do need such files and if so what they should look like. If so I'll add them using a separate PR.

File: README.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ row[1] = "X"
 row #=> [" ", "X", " "]
 ```
 
-*Note: You should not create a multi-dimensional or nested array. The array should be flat and have only string values.*
+*Note: You should create a simple array with one element for each position on the board. There are some more complex ways you could solve this with an array for each row of the board saved within the array for the board (a nester or multi-dimensional array). Don't do that - it'll just make your code harder to work with for now.
```
Comment (pletcher) [inline README.md]: Nit: "nester" should be "nested," right?

**Comment by pletcher** [comment]
@PeterBell thanks for this! Those files are indeed important, so it'd be great if we could add them soon. There's a gem you can install: `gem install learn_writer` (actually, you might want to install my fork: git@github.com:pletcher/learn-write.git at the `add-new-files-to-gemspec` branch, since the gem hasn't been updated yet), and then a command that you can run from the repository root (`learn-write`) to add all of those files.

**Comment by PeterBell** [comment]
This is a proposed solution to https://github.com/learn-co-curriculum/ttt-2-board-rb/issues/2

**Comment by pletcher** [comment]
Just went ahead and fixed the typo; looks good! We can add the license etc. separately if you'd prefer.

**Pull request merged**